### ISSUE TITLE
Berryboot Partial Update Finally

### ISSUE THREAD

**Issue by actraiser30** [maxnet/berryboot#751]
Hello There im sharing a Partial Updated Berryboot Version to keep the Project Somehow Alive!!!

https://easyupload.io/cj7iw2

Changelog: 

Added support for arm_boost so newer pi`s 4 run at 1800mhz (Revisions v1.2 and up)
Slight faster boot/install
Slight better usb/hdd/sdd boot compatibility/speed
Slight faster memory card access time/speed
Updated Boot Files
Support for Pi4 Rev1.5 (untested)
etc.

Enjoy!!!

**Comment by agoldcheidt** [comment]
This is a great contribution to this project. Mirror on: https://berryboot.alexgoldcheidt.com/go/berrytest-server

**Comment by actraiser30** [comment]
Updated!!!!

**Comment by DarthRaver** [comment]
Can i install this update to an existing installation on a raspberry 400 without damaging my images? If so, how to do it?

**Comment by actraiser30** [comment]
Yes you can just copy the files to the boot partition of your card press
yes to overwrite all files backup your old files before that

Στις Κυρ, 16 Ιουλ 2023, 02:05 ο χρήστης DarthRaver ***@***.***>
έγραψε:

**Comment by mvdm85** [comment]
Thank you so much for this!  I can now finally use the latest OSMC on Berryboot on my Pi4.  The past version of Berryboot would boot up the latest OSMC and the UI would work fine, but when it actually wouldn't play anything.  After a lot of messing around I finally figured out that the kernel was too old.

Only suggestion that I can make is please make the download a bit more clear on https://berrybeta.alexgoldcheidt.com/berryboot/dimitris-unofficial-berryboot-update/
Maybe clearly state it in the changelog?
The file name of the latest version makes it appear that it is much older.  I tried the June 28 2023 version which caused a lot of fails during boot for all OS's.  I then tried 12 September 2023 which seems to work great so far!

Thanks again!  And great software!

**Comment by actraiser30** [comment]
Feel free to donate if you want 😄

**Comment by icountu** [comment]
Is there any possibility of Pi5 release?

**Comment by actraiser30** [comment]
Unfortunately not,the creator floris bros that also has, Pinns that is totally crap and full of issues and disadvantages,compared to Berryboot is not doing anything to update it, over 3 years now,still don't understand why he choose Pinns over berryboot.

**Comment by maxnet** [comment]
actraiser, I am not involved with PINN.
It does is derived from NOOBS, whose initial version I did program ages ago as a contractor for Raspberry Pi Ltd.

May update Berryboot some time, but have some other stuff to work on.

**Comment by actraiser30** [comment]
No problem if you are or not,,just a version update for pi5 if possible will be great.thx

**Comment by icountu** [comment]
> actraiser, I am not involved with PINN. It does is derived from NOOBS, whose initial version I did program ages ago as a contractor for Raspberry Pi Ltd.
> 
> May update Berryboot some time, but have some other stuff to work on.

Thanks and Please do it asap.

**Comment by actraiser30** [comment]
Noobs is out of the picture,too slow,installing an os can take forever,tons of bugs and conflicts and unstable.

Berryboot on the other side is fast,stable, you can have hundreds of oses, and you can convert your own very easy.

If I knew how to make an up to date version il do it my self unfortunately building/compiling is not working anymore.

**Comment by agoldcheidt** [comment]
> May update Berryboot some time, but have some other stuff to work on.

@maxnet good day. Happy Holidays, let me know how I can help, a lot of people are requesting new images for Berryboot on my website and a lot more are requesting support on how they can update the kernel. We have been working on the same goal but different directions, just to let you know, I can maintain your Berryboot custom images up-to-date and I'm willing to any possible merge of my work with yours. The Berryboot project seems to be run by only you, but it doesn't have to be that way.

Keep that in mind.
Have a great day.
